compiler/luci/service/src/Nodes/CircleBatchMatMul.cpp

```diff
@@ -30,4 +76,72 @@ luci::CircleNode *CloneNodeLet<CN::ABC>::visit(const luci::CircleBatchMatMul *no
   return cloned;
 }
 
+// BatchMatMulV2 supports broadcasting in the batch dimensions(BatchMatMul doesn't)
+// TODO Distinguish BatchMatMul and BatchMatMulV2
+loco::TensorShape sinf::Algorithm::visit(const luci::CircleBatchMatMul *node)
```

Comment:
@zetwhite , please check https://github.com/Samsung/ONE/pull/13780#pullrequestreview-2265214545

can you apply here too? or as getting approvals takes time and you've go all ACKs, 
apply in another PR?

Reply:
if go with another PR, you can apply changes to all Ops (I think there aren't too many)

Reply:
I'd like to merge this PR as is. 
If fine, I'll make another PR to fix it.

Reply:
> you can apply changes to all Ops (I think there aren't too many) 

Oh, I see. I apply it at once.